src/go/docker-credential-none/cmd/erase.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+var eraseCmd = &cobra.Command{
+	Use:   "erase",
+	Short: fmt.Sprintf("Update the auths in ~/.docker/%s based on the data written to stdin.", configFileName),
```

Comment:
s/auths/???/

I kind of want to request a more detailed description on the incoming data, but I guess this is just the docker-credential-helper format…

Reply:
Improved. But keep in mind this utility is intended for internal use, although users might use it under guidance of QA, or if they're curious.

Reply:
I modified the `Long` field in `root.go` to specify that this helper stores the credentials in the same format that docker uses when no `credsStore` is specified.  Viewed when typing `.../docker-credential-helper -h`